pages/02.Documentation/10.metakube-webterminal/default.en.md

```diff
@@ -8,10 +8,16 @@ taxonomy:
 ---
 
 Every MetaKube cluster comes with a web-terminal directly in the MetaKube web interface that gives command line access to the cluster directly from the browser.
-To access the dashboard just click on the `Web Terminal` link on the cluster detail page. This bash terminal runs on Alpine Linux in the background and comes installed with all the necessary tools to interact with your cluster:
+To access the dashboard just click on the `Web Terminal` link on the cluster detail page:
+
+![MetaKube Dashboard](metakube-dashboard.png)
```

Comment:
This should be an image of the webterminal

Reply:
Added it further below